ocw/lib/EC2.py

```diff
@@ -76,6 +77,23 @@ def all_clusters(self):
             [clusters.append(cluster) for cluster in response['clusters']]
         return clusters
 
+    def delete_outdated_snapshots(self):
+        cleanup_ec2_max_snapshot_age_days = int(self.cfgGet('cleanup', 'ec2-max-snapshot-age-days'))
```

Comment:
`int()` isn't needed. You will always get the same type as your `default` value has.